tla-types/src/test/scala/at/forsyte/apalache/tla/types/TestTypeAnnotationPass.scala

```diff
@@ -81,12 +81,21 @@ class TestTypeAnnotationPass extends FunSuite with TestingPredefs with BeforeAnd
     testFromFile( "realSpecs/Paxos.tla" )
   }
 
-//  test( "Test: Blockchain" ) {
+//  ignore( "Test: Blockchain" ) {
 //    testFromFile( "realSpecs/BlockchainP1.tla" )
 //  }
 //
-//  test( "Test: test7.tla (Lightclient)" ) {
+//  ignore( "Test: test7.tla (Lightclient)" ) {
 //    testFromFile( "test7.tla" )
 //  }
```

Comment:
Shall we exclude these tests from the base? I think they should be the integration tests.